bin/hooks/pre-commit

```diff
@@ -5,18 +5,20 @@
 # @package WP Enforcer
 
 # Determine if we have a phpcs.xml file
-if [[ -f ./phpcs.xml ]]; then
-	standard="--standard=./phpcs.xml"
+if [[ "phpcs" -eq "{phpcs_path}" && 0 -lt $(phpcs -i | grep -woe {phpcs_standard} | wc -w) ]]; then
```

Comment:
For consistency's sake, it seems to me that `phpcs` in `phpcs -i | grep -woe {phpcs_standard} | wc -w` ought to be using the `phpcs_path` placeholder.

Reply:
I see the confusion here.

phpcs_path means `which phpcs` in other words that does mean, the path of the phpcs bin we want to use.
phpcs_standard could be, a file, a stardard it self as, WordPress-Extra or WordPress-VIP

That check works for a global environment phpcs bin and global environment WordPress standard.

On the README.md file I added this example:
This one is the main reason of the PR
```json
{
	"scripts": {
		"post-install-cmd": [
			"wp-enforcer <path/to/dir>"
		],
		"post-update-cmd": [
			"wp-enforcer <path/to/dir>"
		]
	}
}
```
This other one is an extra feature that I thought.
```json
{
	"scripts": {
		"post-install-cmd": [
			"wp-enforcer global WordPress-Core"
		],
		"post-update-cmd": [
			"wp-enforcer global WordPress-Core"
		]
	}
```

The point of this PR is for the case when we have our phpcs.xml in a folder level below our .git folder.

That line you mention actually is an extra feature I added, to use global phpcs installation with global WordPress Standard installation. It does confirm that the global phpcs installation has the standard you want to use.

We could use a global phpcs and a lower folder level phpcs.xml
```json
{
	"scripts": {
		"post-install-cmd": [
			"wp-enforcer global path/to/phpcs.xml"
		],
		"post-update-cmd": [
			"wp-enforcer global path/to/phpcs.xml"
		]
	}
```

Reply:
Hi @killua99 

Thanks for taking the time to write up an in-depth explanation of the change in functionality.

For my part, understanding what the code does and how it works was not my problem.

> For consistency's sake

That was the operative part of my comment. I understand that the call to `phpcs` there is for global installations and it won't run that block of code unless configured to use the global installation. My comment was that we shouldn't be hard-coding the location of `phpcs`, even for blocks of code that only get run in a specific context. We should be using a consistent reference for all uses of the PHPCS binary. My reasoning for this is that if users manually edit the hook script (not supported and futile, but bound to happen) there won't be any confusion about why we're trying to use PHPCS from different locations.

Please let me know if my clarification makes sense.

Reply:
Sorry I didn't catch it at first.

So if I'm correct, you would like to see something like this:

`if [[ 0 -lt $("{phpcs_path}" -i | grep -woe {phpcs_standard} | wc -w) ]]; then`

🤔 I might want to test it before confirm might not cause any trouble. Do you believe that would work ok?